src/Utils/TimestampFormatter.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\PageApprovals\Utils;
+
+use Language;
+use MWTimestamp;
+
+class TimestampFormatter {
+
+	public static function formatTimestamp( ?int $timestamp, Language $language ): string {
+		if ( !$timestamp ) {
+			return '';
+		}
+		return $language->getHumanTimestamp( new MWTimestamp( $timestamp ) );
+	}
+
+}
```

Comment:
This "class" (scare quotes because it only as a _static_ method) does not justify itself. Especially if you extract the details into a dedicated private function in the first place.

Reply:
Its true. There really isn't any need for a separate class. Its being used only in 2 places.